src/SharpBrick.PoweredUp.BlueGigaBLE/BlueGigaBLEStatus.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Extensions.Logging;
+
+namespace SharpBrick.PoweredUp.BlueGigaBLE
+{
+    public class BlueGigaBLEStatus
+    {
+        public const UInt16 STATE_STANDBY = 0;
```

Comment:
Now this combination is definitely Java style ;) C# would use `StateStandby` in an enum with a type.

````cs
enum BleStates : ushort
{
    StandBy = 0x0000,
    Scanning = 0x001,
    // ...
}

Reply:
Changed to C#-style :-)